README.md

```diff
@@ -235,6 +235,12 @@ Parse Dashboard supports adding an optional background color for each app, so yo
 }
 ```
 
+## Custom branding
+
+- You can set `customBrandIcon` with relative path from `iconsFolder`. This could be your custom logo which will appear on login screen and on the top of sidebar. Icon should be square (same height and width), SVG or PNG with transparent background. 
```

Comment:
```suggestion
- You can set `customBrandIcon` with a relative path from `iconsFolder`. This could be your custom logo which will appear on the login screen and on the top of the sidebar. Icon should be square (same width and height), SVG or PNG with transparent background.
```